<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-05-12T05:39:16Z" name="Default suite" finished-at="2019-05-12T05:39:38Z" duration-ms="22762">
    <groups>
    </groups>
    <test started-at="2019-05-12T05:39:16Z" name="Default test" finished-at="2019-05-12T05:39:38Z" duration-ms="22762">
      <class name="Amazon.TestScreenshot">
        <test-method is-config="true" signature="setup()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:16Z" name="setup" finished-at="2019-05-12T05:39:21Z" duration-ms="5797" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Takescreenshot()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:21Z" name="Takescreenshot" finished-at="2019-05-12T05:39:21Z" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Takescreenshot -->
        <test-method is-config="true" signature="Teardown()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:21Z" name="Teardown" finished-at="2019-05-12T05:39:22Z" duration-ms="628" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:22Z" name="setup" finished-at="2019-05-12T05:39:29Z" duration-ms="7023" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Takescreenshot1()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:29Z" name="Takescreenshot1" finished-at="2019-05-12T05:39:29Z" duration-ms="14" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: expected:<false> but was:<true>
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.failNotEquals(Assert.java:282)
at junit.framework.Assert.assertEquals(Assert.java:64)
at junit.framework.Assert.assertEquals(Assert.java:149)
at junit.framework.Assert.assertEquals(Assert.java:155)
at Amazon.TestScreenshot.Takescreenshot1(TestScreenshot.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Takescreenshot1 -->
        <test-method is-config="true" signature="Teardown()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:30Z" name="Teardown" finished-at="2019-05-12T05:39:31Z" duration-ms="683" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:31Z" name="setup" finished-at="2019-05-12T05:39:37Z" duration-ms="5996" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Takescreenshot2()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:37Z" name="Takescreenshot2" finished-at="2019-05-12T05:39:37Z" duration-ms="1" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: expected:<false> but was:<true>
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.failNotEquals(Assert.java:282)
at junit.framework.Assert.assertEquals(Assert.java:64)
at junit.framework.Assert.assertEquals(Assert.java:149)
at junit.framework.Assert.assertEquals(Assert.java:155)
at Amazon.TestScreenshot.Takescreenshot2(TestScreenshot.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Takescreenshot2 -->
        <test-method is-config="true" signature="Teardown()[pri:0, instance:Amazon.TestScreenshot@10aa41f2]" started-at="2019-05-12T05:39:38Z" name="Teardown" finished-at="2019-05-12T05:39:38Z" duration-ms="705" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- Amazon.TestScreenshot -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
